src/api.js

```diff
@@ -4,13 +4,14 @@ const reviewsURL = axios.create({
     baseURL: 'https://gamesmarsncapi.onrender.com/api'
 })
 
-export const fetchAllReviews = (category) => {
-    if(category === 'all' || category === undefined){
+export const fetchAllReviews = (categorySearchParams, orderSearchParams='desc', sortBySearchParams='title') => {
```

Comment:
This should be refactored to make sure of Axios params rather than relying on string manipulation. Axios has the ability to ignore null queries, preventing them from being added to the endpoint. Take a look at the Axios section of NC Notes if you need a refresher